src/couch/src/couch_rand.erl

```diff
@@ -0,0 +1,57 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_rand).
+
+
+-export([
+    uniform/0,
+    uniform/1
+]).
+
+
+-ifdef(NORANDMODULE).
+
+
+uniform() ->
+    maybe_set_random_seed(),
+    random:uniform().
+
+
+uniform(N) ->
+    maybe_set_random_seed(),
+    random:uniform(N).
+
+
+maybe_set_random_seed() ->
```

Comment:
both rand:uniform() and random:uniform() docs claim they will set a seed if one isn't set, so why are we doing it?

'If a process calls uniform/0 or uniform/1 without setting a seed first, seed/0 is called automatically.'

Reply:
because random:uniform() uses a constant seed https://github.com/erlang/otp/blob/1526eaead833b3bdcd3555a12e2af62c359e7868/lib/stdlib/src/random.erl#L45 so every process will effectively go through the same sequence of random numbers, which often is not helpful. In a few places we had to set a random seed in a few places to fix that behavior. 

The new module https://github.com/erlang/otp/blob/418a06c1fa14d144abb95d6e2030e49d074cf6e6/lib/stdlib/src/rand.erl#L243 uses a random seed, that's the behavior we want usually.

So here we basically try to emulate the behavior of rand module.